qdrant-landing/content/documentation/overview/_index.md

```diff
@@ -9,15 +9,18 @@ aliases:
 
 ![qdrant](https://qdrant.tech/images/logo_with_text.png)
 
-Vector databases are a relatively new way for interacting with abstract data representations 
-derived from opaque machine learning models such as deep learning architectures. These 
-representations are often called vectors or embeddings and they are a compressed version of 
-the data used to train a machine learning model to accomplish a task like sentiment analysis, 
-speech recognition, object detection, and many others.
-
-These new databases shine in many applications like [semantic search](https://en.wikipedia.org/wiki/Semantic_search) 
-and [recommendation systems](https://en.wikipedia.org/wiki/Recommender_system), and here, we'll 
-learn about one of the most popular and fastest growing vector databases in the market, [Qdrant](https://qdrant.tech).
+Vector databases can help you interact with machine learning models. As a compressed version of data used to train such models, it can help you accomplish tasks like:
+
+- Sentiment analysis
+- Speech recognition
+- Object detection
```

Comment:
We could link the example tasks to existing Qdrant use cases (taken from https://qdrant.tech/solutions/) 

```suggestion
- [Similar Image Search](https://food-discovery.qdrant.tech/)
- [Semantic Text Search](https://demo.qdrant.tech/)
- [Anamoly Detection](https://qdrant.tech/articles/detecting-coffee-anomalies/)
```

Reply:
Alternatively, if these examples are not appropriate, then we could remove them. After all, this is an introduction. Do either of these options make sense, @generall ?